tests/v2/testlib/waku2.nim

```diff
@@ -0,0 +1,58 @@
+import
+  std/options,
+  stew/byteutils,
+  libp2p/switch,
+  libp2p/builders
+import
+  ../../../waku/v2/protocol/waku_message,
+  ./common
+
+export switch
+
+
+# Switch
+
+proc generateEcdsaKey*(): PrivateKey =
+  PrivateKey.random(ECDSA, rng[]).get()
+
+proc generateEcdsaKeyPair*(): KeyPair =
+  KeyPair.random(ECDSA, rng[]).get()
+
+proc generateSecp256k1Key*(): PrivateKey =
+  PrivateKey.random(Secp256k1, rng[]).get()
+
+proc generateKey*(): PrivateKey =
```

Comment:
Perhaps remove `generateKey()` and use generateEcdsaKey and generateSecp256k1Key?

Reply:
Honestly, this was to simplify the PR. But, as you wish 🤷🏼‍♂️